src/cpuid.cpp

```diff
@@ -46,7 +46,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <cstring> // for std::memset
 #endif
 
-#if TORRENT_HAS_ARM
+#if TORRENT_HAS_ARM && TORRENT_HAS_ARM_NEON
```

Comment:
@aldenml does this seem reasonable? as far as I can tell, this function is only used if ``TORRENT_HAS_ARM_NEON``, and in my cross compiler for ``arm-linux-gnueabihf`` does not have this header